TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -593,6 +605,8 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 				}.observes('aspectRatioWidth', 'aspectRatioHeight').on('init'),
 
 				_aspectRatioDidChange: function () {
+					parent.set('_showOriginalImageInInspector', false);
+					parent.set('_showOriginalImageInInspector', true);
```

Comment:
ah, well observers are not supposed to be triggered like that.. instead just call `that. _swapPreviewWithOriginal();` to trigger the functionality if needed when not changing the observed value..